apps/src/sites/studio/pages/levels/show.js

```diff
@@ -61,14 +61,21 @@ function initPage() {
 
   if (hasScriptData('script[data-aitutordata]')) {
     const aiTutorData = getScriptData('aitutordata');
-    const {levelId, type, hasValidation, isAssessment, isProjectBacked} =
-      aiTutorData;
+    const {
+      levelId,
+      type,
+      hasValidation,
+      isAssessment,
+      isProjectBacked,
+      levelSpecificPrompt,
+    } = aiTutorData;
     const level = {
       id: levelId,
       type: type,
       hasValidation: hasValidation,
       isAssessment: isAssessment,
       isProjectBacked: isProjectBacked,
+      levelSpecificPrompt,
```

Comment:
ohh can we shortcut like this or do we need `levelSpecificPrompt: levelSpecificPrompt`? If the short form works, let's make the fields consistent.